### PR TITLE
feat(ProjectMigration): script to migrate a project field to new value

### PR DESCRIPTION
**Pull request by JaideepPalit**
Migration script to migrate a project field to new value.
Following variables needs to be updated based on requirement
`fieldName` = "field_to_be_migrated"
`oldValue` = "old_value"
`newValue` = "new_value"

For updating ClearingTeam Field in Project - `clearing.teams` property in `sw360.properties` file should be updated accordingly

Signed-off-by: Jaideep Palit <jaideep.palit@siemens.com>

### CODE REVIEW

File: scripts/migrations/015_migrate_project_field_value.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/python
```
Comment (maxhbr) [inline scripts/migrations/015_migrate_project_field_value.py]: This script needs to be mentioned and explained in the `README.md`

Comment (JaideepPalit) [reply scripts/migrations/015_migrate_project_field_value.py]: Thanks @maxhbr for suggestion.
Have mentioned and explained the script in README.md

**Comment by maxhbr** [comment]
This script looks more like a utility instead of a migration script which compensates database schema changes. Maybe move it to a different folder?

**Comment by JaideepPalit** [comment]
Moved script to a different folder.

**Pull request merged**